alvr/client_openxr/src/interaction.rs

```diff
@@ -558,6 +582,7 @@ pub fn get_hand_data(
     platform: Platform,
     reference_space: &xr::Space,
     time: Duration,
+    future_time: Duration,
```

Comment:
I mean clippy isn't wrong, but I also can't think of a good way to solve this lint, maybe just disable the warning there

Reply:
I.e. `expect`

Reply:
yup, that's a prime spot for expect